### ISSUE TITLE
@clayui/css Chrome adds focus outline on elements with [tabindex="-1"] when clicked

### ISSUE THREAD

**Issue by pat270** [liferay/clay#3436]
## What is your proposal?
 From MDN:
> A negative value (usually tabindex="-1") means that the element is not reachable via sequential keyboard navigation, but could be focused with Javascript or visually by clicking with the mouse. It's mostly useful to create accessible widgets with JavaScript.

We should disable focus outline on `tabindex="-1"` elements by default by adding a rule-set that removes focus outline on any element with a negative tabindex `[tabindex^='-'] { outline: 0; }`. We can redo the focus outline if we need that behavior.

## Why would adopting this proposal be beneficial?

This causes focus outline on click for `a.disabled` elements that shouldn't have them. It's also an issue in places like https://clayui.com/docs/components/modal.html, where clicking the modal dialog shows a focus border.
